### PR TITLE
[0.4] Docs: Add KeePass database 1->2 migration guide

### LINKED ISSUE #1970: Document KeePass migration in Tails upgrade docs
# Feature request

## Description

As pointed out by @freddymartinez9 in https://github.com/freedomofpress/securedrop/issues/1551#issuecomment-314818161, the Tails 2.x -> 3.x upgrade guide does _not_ include specific instructions for migrating the KeePass database. It should! Otherwise Admins and Journalists will overlook the manual intervention required (see #1919 and related), silently removing access to site-specific secrets in the passphrase manager.

## User Stories

As an Admin, I want a clearly written list of specific actions I must take in order to upgrade my organization's SecureDrop instance to 0.4. If manual actions are required to ensure access to KeePass databases, tell me what I need to do, so I can do it.

### PR DESCRIPTION
**Pull request by redshiftzero**
## Status

Ready for review

## Description of Changes

Fixes #1970, adds description to Tails 3 migration guide about updating password databases to `kdbx` format.

## Testing

Check that the steps here are accurate. Provision docs locally, check this renders fine

## Deployment

Docs only

### If you made changes to documentation:

- [x] Doc linting passed locally

### CODE REVIEW

File: docs/upgrade_to_tails_3x.rst [hunk 1]
```diff
@@ -227,7 +227,21 @@ Workstation* drives:
 
 .. |Terminal| image:: images/terminal.png
 
-5. Verify the Upgrades
+7. Upgrade KeePassX Database
```
Comment (dachary) [inline docs/upgrade_to_tails_3x.rst]: 5. here

**Comment by dachary** [comment]
I verified the steps successfully, using the attached password file in the old format, with master password foobar : http://dachary.org/loic/oldformat.kdb

:+1: once the typo is fixed

**Comment by redshiftzero** [comment]
Thanks for the review @dachary 😄  Fixed that typo and repushed

**Comment by redshiftzero** [comment]
Excellent, thanks for review @conorsch @dachary. Merging!

**Pull request merged**